modules/authentication/server-ts/social/linkedIn/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Express } from 'express';
+import passport from 'passport';
+import { Strategy as LinkedInStrategy } from '@sokratis/passport-linkedin-oauth2';
+import settings from '../../../../../settings';
+import AuthModule from '../AuthModule';
+
+const { clientID, clientSecret, callbackURL, enabled } = settings.auth.social.linkedin;
+
+interface AppContext {
+  social: any;
+}
+
+const middleware = (app: Express, appContext: AppContext) => {
+  if (!enabled || __TEST__) {
+    return false;
+  }
+
+  app.use(passport.initialize());
+
+  app.get('/auth/linkedin', (req, res, next) => {
```

Comment:
Social auth controllers initialize in another from others way.

Reply:
Reworked.